packages/mobile/src/pages/chat/chat-page.tsx

```diff
@@ -541,12 +541,14 @@ const ChatPage = memo(({ route, navigation }: ChatProp) => {
             openURL('https://ravenhosting.cloud/report/content');
           },
         },
-        {
-          label: `${t('Block this user')}`,
-          onPress: () => {
-            openURL(`${host}/owner/connections/${filteredRecipients?.[0]}/block`);
-          },
-        },
+        isGroupChat
+          ? undefined
```

Comment:
I know that it makes no sense to block all in a group channel. But if we wanna make sure we have enough features to reply to Apple Review, we ought to replace this to adding a block onto the message items themselves? What do you think?